.circleci/config.yml

```
@@ -35,6 +35,11 @@ jobs:
             # fallback to using the latest cache if no exact match is found
             - v1-dependencies-
 
+      # TODO PREAPPS-7520
+      - run:
+          name: Set correct npm version
+          command: sudo npm install -g npm@6.14.15
```

we can link to just the major version
```suggestion
          command: sudo npm install -g npm@6
```

---

Done